NuEVI/NuEVI.ino

```diff
@@ -263,7 +263,9 @@ void setup() {
 
   // if stored settings are not for current version, or Enter+Menu are pressed at startup, they are replaced by factory settings
   
-  if (((readSetting(VERSION_ADDR) != VERSION) && (readSetting(VERSION_ADDR) < 24)) || (!digitalRead(ePin) && !digitalRead(mPin))) {
+  uint16_t settingsVersion = readSetting(VERSION_ADDR);
+
+  if (((settingsVersion != VERSION) && (settingsVersion < 24)) || (!digitalRead(ePin) && !digitalRead(mPin)) || (settingsVersion == 0xffffu)) {
```

Comment:
The uninitialised EEPROM contains the value 0xffff, and since that is > 24, this if statement was never true when that happened.
Note that a new Teensy 3.2 contains code that writes to the beginning of EEPROM when started, so in reality, a value of 0xffff was very unlikely.